### PR TITLE
Node nginx

### PR DESCRIPTION
**Pull request by alikhajeh1**

### CODE REVIEW

File: node/8/nginx.conf [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    gzip_disable msie6;
+
+    # Set client's real IP
+    real_ip_recursive on;
+    real_ip_header X-Forwarded-For;
+    set_real_ip_from 10.0.0.0/8;
+
+    location / {
+        proxy_pass http://127.0.0.1:3000;
+        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
```
Comment (alikhajeh1) [inline node/8/nginx.conf]: I think we want to remove this from here and the Ruby nginx.conf - aye?

File: node/8/start.sh [hunk 2]
```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+if [ "$ENTRYPOINT" = "workers" ]
+then
+  echo Starting workers
+  # Put any commands here to start background workers
```
Comment (alikhajeh1) [inline node/8/start.sh]: Have left this here in case users need other entrypoints for workers

File: node/8/nginx.conf (hunk 1 above)
Comment (aliscott) [reply node/8/nginx.conf]: Yeah I don't think we need it

File: node/8/Dockerfile [hunk 3]
```diff
@@ -7,20 +7,28 @@ ENV TZ=Asia/Tehran \
...
 RUN apk update && \
     # Install required packages - you can find any additional packages here: https://pkgs.alpinelinux.org/packages
-    apk add tzdata curl bash ca-certificates rsync \
+    apk add tzdata curl bash ca-certificates rsync supervisor nginx \
             python python-dev py-pip build-base libpng-dev autoconf automake nasm libtool && \
     # Set the timezone
     cp /usr/share/zoneinfo/${TZ} /etc/localtime && \
     echo "${TZ}" > /etc/timezone && \
+    # Set the nginx config
+    sed -i "/user nginx;/c #user nginx;" /etc/nginx/nginx.conf && \
```
Comment (aliscott) [inline node/8/Dockerfile]: Do we need this?

Comment (aliscott) [reply node/8/Dockerfile]: Ignore me, I didn't notice we were commenting this out, which is what we want.

**Pull request merged**